games-arcade/rocksndiamonds/additional-files/rocksndiamonds.rdef

```diff
@@ -0,0 +1,39 @@
+// Rocks 'n' Diamonds icon by Stephanie Fu
+
+resource app_signature "application/x-vnd.rocksndiamonds";
+
+resource app_flags B_EXCLUSIVE_LAUNCH;
+
+resource app_version {
+	major = 0,
+	middle = 0,
+	minor = 0,
```

Comment:
looks like I needed to submit my comments before they are shown ;)

PS, I took some time to create one for the 4.0.0.0.rc3 a while back, maybe that could be used as a base? IIRC there were some isseus with that one, hence why I didn't creat a pr back then :)
https://github.com/Begasus/haikuports/tree/rocksndiamonds